pkg/updater/gcs.go

```diff
@@ -529,6 +529,9 @@ func overallCell(result gcsResult) Cell {
 // ElapsedKey is the key for the test duration metric.
 const ElapsedKey = "test-duration-minutes"
 
+// TestMethodsElapsedKey is an automatically-added metric for sponge.
```

Comment:
nit: Would match this to "ElapsedKey" above for consistency.

Reply:
done